### DISCUSSION TITLE
Only for those who want to apply

### DISCUSSION THREAD

**Discussion by inyourface34456** [inyourface34456/bookmarklets-website#3] [Want to apply?]
You can apply by metonling me in the comments of this post.

**Comment by PSYclownYT** [comment]
@inyourface34456 I am working on a bookmarklet pack myself, and I think that I could help you with this. here is the page for my pack, I have been working on this for the past year or so.
https://github.com/topics/eliotsepicbookmarklets

**Reply by inyourface34456** [reply]
I was looking through it, and it seems that a lot of them are mine (not all, but most).  It seems that you reorganized my bookmarklet import and added a few of your own.  It also appears that some of them load JavaScript from a website rather than in the bookmarklet itself, which is bad practice.  The only issue on the repo is you saying that the code is obfuscated and then you closed it and reopened it very quickly.  Your repo also only had its first commit on March 16th, 2023.  That is way shorter then mine has been online.  In short, I am rejecting you.

**Reply by PSYclownYT** [reply]
I have never seen yours in my life

**Reply by PSYclownYT** [reply]
i also didnt have it on github the whole time

**Reply by inyourface34456** [reply]
I it appears that a lot of the bookmarklets are in a very similar order and format as mine. You even have the same bookmarklets that I created, the text shianigans folder, which is an original creation of mine. You also have the operational version of favlets, which cannot be found anywhere else (as far as I know).  I only see a handful of your contributions in the file, maybe a few dozen original/ gathered by you bookmarklets, and you also restructured the entire thing.

**Reply by PSYclownYT** [reply]
most of them are from here, https://github.com/dragon731012/-WORKING-bookmarklets-and-games
also i did forget to cite on the repo, Thanks for the reminder.

**Reply by PSYclownYT** [reply]
I did this myself. if you look at the first commit, it is VERY small, and contains none of yours. I am fine with the rejection, and thank you for the citation feedback.
